src/master-playlist-controller.js

```diff
@@ -599,13 +612,55 @@ export class MasterPlaylistController extends videojs.EventTarget {
   }
 
   /**
+   * fill our internal list of Subtitle Tracks with data from
+   * the master playlist or use a default
+   *
+   * @private
+   */
+  fillSubtitleTracks_() {
+    let master = this.master();
+    let mediaGroups = master.mediaGroups || {};
+
+    for (let mediaGroup in mediaGroups.SUBTITLES) {
+      if (!this.subtitleGroups_.groups[mediaGroup]) {
+        // this.subtitleGroups_.groups[mediaGroup] = { unforced: [], forced: {} };
+        this.subtitleGroups_.groups[mediaGroup] = [];
+      }
+
+      for (let label in mediaGroups.SUBTITLES[mediaGroup]) {
+        let properties = mediaGroups.SUBTITLES[mediaGroup][label];
+
+        if (!properties.forced) {
+          this.subtitleGroups_.groups[mediaGroup].push(
+            videojs.mergeOptions({ id: label }, properties));
+
+          if (typeof this.subtitleGroups_.tracks[label] === 'undefined') {
+            let track = this.tech_.addRemoteTextTrack({
+              id: label,
+              kind: 'subtitles',
+              enabled: false,
+              language: properties.language,
+              label
+            }, true).track;
+
+            this.subtitleGroups_.tracks[label] = track;
+          }
+        }
+      }
+    }
+  }
+
+  /**
    * Call load on our SegmentLoaders
    */
   load() {
     this.mainSegmentLoader_.load();
     if (this.audioPlaylistLoader_) {
       this.audioSegmentLoader_.load();
     }
+    if (this.subtitlePlaylistLoader_) {
+      this.subtitleSegmentLoader_.load();
```

Comment:
from #1043 
@imbcmdth: At some point I feel like it might just be nice to have an array of active segment loaders. Then we can just do this.segmentLoaders_.forEach((l) => l.load());... not necessary now but maybe a good idea to think about soon.